devops/git/workflow.md

```diff
@@ -43,7 +44,7 @@ Reviewers must directly link to the relevant section of the documentation when r
 re-wording the documentation in their own words.
 
 Only the person who created the pull request or admin should merge it. 
-Even though it may of been accepted by the reviewers the person who created it might of noticed a change after the fact or admins might have a reason for delaying the merge.
+Even though it may of been accepted by the reviewers the person who created it might have noticed a change after the fact or admins might have a reason for delaying the merge.
```

Comment:
This would be resolved when #122 gets merged in. This'll probably cause conflicts now 😅

Reply:
If it's the exact same change, git should merge is cleanly. :crossed_fingers: